windows/src/desktop/setup/locale/pt-PT/strings.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <string name="ssLanguageName" comment="User interface language name in the UI language">Português de Portugal</string>
-  <string name="ssLanguageNameInEnglish" comment="User interface language name in English">Portuguese</string>
```

Comment:
Do we only need this in the en strings.xml ? Makes sense to me if that is the case, just checking.